server/app/modules/CloudStorageModule.java

```diff
@@ -18,68 +23,40 @@
 
 /** Configures and initializes the classes for interacting with file storage backends. */
 public class CloudStorageModule extends AbstractModule {
-
-  private static final String AZURE_STORAGE_CLASS_NAME =
-      "services.cloud.azure.AzureApplicantStorage";
-  private static final String AWS_STORAGE_CLASS_NAME = "services.cloud.aws.AwsApplicantStorage";
-
-  private final Environment environment;
   private final Config config;
 
-  public CloudStorageModule(Environment environment, Config config) {
-    this.environment = checkNotNull(environment);
+  public CloudStorageModule(Environment unused, Config config) {
```

Comment:
question (non-blocking): Why pass in unused here? I haven't seen this pattern before so I'm curious.

Reply:
I think either Play or Guice requires that the `Environment` gets passed in? If I remove it I get this error: `play.api.PlayException: No valid constructors[Module [modules.CloudStorageModule] cannot be instantiated.]`. I couldn't find any actual documentation in Play about this though. I added a comment, but happy to look more into why it's required in the first place if we think that's worthwhile.

Reply:
weird! thanks for adding the comment, i think that's good enough for now.